src/Microsoft.Diagnostics.EventFlow.Outputs.HttpOutput/Configuration/HttpOutputConfiguration.cs

```diff
@@ -31,7 +35,8 @@ public HttpOutputConfiguration DeepClone()
                 Format = this.Format,
                 HttpContentType = this.HttpContentType,
                 BasicAuthenticationUserName = this.BasicAuthenticationUserName,
-                BasicAuthenticationUserPassword = this.BasicAuthenticationUserPassword
+                BasicAuthenticationUserPassword = this.BasicAuthenticationUserPassword,
+                Headers = this.Headers
```

Comment:
`Headers = new Dictionary<string, string>(this.Headers)`

It is a "deep clone", so you do not want to share the same headers set between the original and the clone

Reply:
Sure, added :)